src/plugins/condition/components/ConditionSet.vue

```diff
@@ -2,9 +2,11 @@
 <div class="c-object-view u-contents">
```

Comment:
Please remove this outer div - `<div class="c-object-view u-contents">`

Reply:
Done.